src/Process/CpuCoreCounter.php

```diff
@@ -24,42 +16,11 @@ public function getNumberOfCpuCores(): int
 			return $this->count;
 		}
 
-		if (!function_exists('proc_open')) {
-			return $this->count = 1;
+		try {
+			return (new FidryCpuCoreCounter())->getCount();
```

Comment:
also the caching into `$this->count` should probably be kept here :)

Reply:
Missed that 🤦